### ISSUE TITLE
API V2 - Update customer

### ISSUE THREAD

**Issue by PapsOu** [betaglop/e-venement#258]
... cannot update « subscribedToNewsletter » field

POST https://e-venement.local/api/v2/customers/123
```json
{
    "id": 123,
    // [..]
    "subscribedToNewsletter":true
}
```
Getting the updated customer with GET https://e-venement.local/api/v2/customers/123, the field subscribedToNewsletter is still false

**Comment by PapsOu** [comment]
More informations : it should be a problem with the inverted value of the subscribedToNewsletter. The mapping between API field and doctrine field use an inverter to set value when « Reading » the customer.

When « Updating » the customer, this inverter is not present.

Could you add the inverter in the save action for this field only ?

**Comment by betaglop** [comment]
There is an other issue concerning the password updating...

**Comment by betaglop** [comment]
Now the updated customer is correctly returned back by the API, conforming the documentation

**Comment by betaglop** [comment]
corrected by commit 80685e6
